class InvalidException(Exception):
    pass


class GracefulExit(Exception):
    pass
